pulpcore/cli/deb/repository.py

```diff
@@ -99,7 +99,7 @@ def repository(ctx: click.Context, pulp_ctx: PulpContext, repo_type: str) -> Non
         "since the last sync. This greately improves re-sync performance in such cases. Disable if "
         "the sync result does not match expectations."
     ),
-    needs_plugins=[PluginRequirement("deb", min="2.20.0")],
+    needs_plugins=[PluginRequirement("deb", min="2.20.0.dev")],
```

Comment:
This version spec is absolutely legit. We could clean it up but also leave it...

Reply:
I squashed the DROP_ME commit, so this change is now part of the single commit.